app/src/main/java/com/example/habitsmasher/HabitEvent.java

```diff
@@ -0,0 +1,96 @@
+package com.example.habitsmasher;
+
+import android.net.Uri;
+
+import com.google.firebase.firestore.PropertyName;
+
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * This is the Habit Event class
+ * Its purpose is to store and retrieve the start date, comments, picture, and location of a habit event
+ */
+public class HabitEvent {
+    private Date _startDate;
+    private String _comment;
+    private Uri _pictureUri;
+    private UUID _id = UUID.randomUUID();
```

Comment:
Since we are transitioning to `UUID`, when do we want to update the `Habit` class to use this as well?

Reply:
Up to you, I can do it on this PR for consistency if you want

Reply:
We should probably address this in the Database Entity PR, since we'll have Habit, HabitEvents etc. extend from it.

Reply:
The database entity PR is already up for review I think. That story doesn't change any of the existing IDs. If this story is changing the IDs, I think the change should **propagate** to the other data stored in the db. This PR is introducing a new way to do IDs, so it should also **modify** the others to make it consistent. Don't want to **block** this PR, so make sure to create a task for this.
- [ ] create a task to change all IDs to UUID